dev/tasks/verify-rc-alma/Dockerfile

```diff
@@ -0,0 +1,8 @@
+# Container image that runs
+FROM almalinux:8.4
+
+# Copy script into container
+COPY setup-AlmaLinux-Rocky-Linux.sh /setup-AlmaLinux-Rocky-Linux.sh
+
+# Execute script
+ENTRYPOINT ["/setup-AlmaLinux-Rocky-Linux.sh"]
```

Comment:
We don't need this file.
We can use `--volume` for using file on host.

Reply:
Ok. Maybe a verification script for a Docker image can be created in a separate issue?